package.json

```diff
@@ -5,16 +5,25 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn workspace @ssi-trust-registry/backend build",
-    "validate": "yarn workspace @ssi-trust-registry/backend validate && yarn workspace @ssi-trust-registry/frontend validate",
+    "fe:build": "yarn workspace @ssi-trust-registry/frontend build",
+    "be:build": "yarn workspace @ssi-trust-registry/backend build",
+    "build": "yarn be:build && yarn fe:build",
```

Comment:
Is there a reason that we don't use `concurrently` for builds? Only validate and start?

Reply:
No particular reason, we could do it too

Reply:
It seems that the build in docker got stuck with concurrently, so I keep it without that for now.